drivers/scheduler/openshift/openshift.go

```diff
@@ -430,6 +590,13 @@ func getChannel(version string) (string, error) {
 		return version, nil
 	}
 
+	versionSplit := strings.Split(version, "-")
```

Comment:
this change is already in master, rebase your branch with master first

Reply:
Cleaned up and updated the PR.